src/services/export.service.ts

```diff
@@ -97,7 +125,42 @@ export class ExportService implements ExportServiceAbstraction {
         }
     }
 
-    async getOrganizationExport(organizationId: string, format: 'csv' | 'json' = 'csv'): Promise<string> {
+    private async getEncryptedExport(format: 'encrypted_json'): Promise<string> {
+        const folders = await this.folderService.getAll();
+        const ciphers = await this.cipherService.getAll();
+
+        if (format === 'encrypted_json') {
```

Comment:
Wouldn't the format already be `encrypted_json`, that seems to be the only accepted value/parameter in the method.

Reply:
yea, i suppose so. I just don't trust the typing sometimes since you can always pass something as type `any` with another value

Reply:
was also trying to leave room for other format type, like the decrypted options.